src/main/java/org/opensearch/knn/index/KNNSettings.java

```diff
@@ -89,8 +88,7 @@ public class KNNSettings {
     public static final Integer KNN_MAX_MODEL_CACHE_SIZE_LIMIT_PERCENTAGE = 25; // Model cache limit cannot exceed 25% of the JVM heap
     public static final String KNN_DEFAULT_MEMORY_CIRCUIT_BREAKER_LIMIT = "50%";
 
-    public static final Integer ADVANCED_FILTERED_EXACT_SEARCH_THRESHOLD_DEFAULT_VALUE = 2000;
-    public static final Integer ADVANCED_FILTERED_EXACT_SEARCH_THRESHOLD_PCT_DEFAULT_VALUE = 10;
```

Comment:
Did we decide to remove percentage check? Just checking as I though we will just add distance computation count condition in previous logic.

Reply:
Yes we decided to remove the %age check.